### PR TITLE
Add Lump Sums

### PR DESCRIPTION
**Pull request by johnwatson484**
Lump sums file spec is identical to BPS file spec. Just with SITI references replaced with LSES plus different scheme and fund codes.

I've also taken the opportunity to address minor instances of tech debt, but nothing too significant beyond the scope of what I wanted to achieve.

### CODE REVIEW

File: FFCPayBatchGenerator/Factory/BPSFactory.cs [hunk 1]
```diff
@@ -30,12 +27,11 @@ public override string GetContent()
 
         for (int i = 0; i < batchSize; i++)
         {
-            sb.AppendLine(string.Format("H^SITI{0}^{1}^C{2}^{3}^{4}^{5}^{6}^GBP",
+            sb.AppendLine(string.Format("H^SITI{0}^{1}^C{2}^{3}^1^{4}^{5}^GBP",
                 invoiceNumber.ToString("D7"),
                 requestNumber.ToString("D3"),
                 invoiceNumber.ToString("D7"),
                 frn,
-                requestNumber == 1 ? 1 : 2,
```
Comment (johnwatson484) [inline FFCPayBatchGenerator/Factory/BPSFactory.cs:38]: This was actually a bug.  This value represents `Pillar` which is always `1` for BPS.

File: FFCPayBatchGenerator/Factory/BatchFactory.cs [hunk 2]
```diff
@@ -1,24 +1,19 @@
 ﻿
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
-namespace FFCPayBatchGenerator.Factory
+namespace FFCPayBatchGenerator.Factory;
```
Comment (johnwatson484) [inline FFCPayBatchGenerator/Factory/BatchFactory.cs:4]: There was a mix of convention between file scoped namespaces and more traditional namespaces declarations.

For consistency I've converted them all to the more modern file scoped namespaces.

File: FFCPayBatchGenerator/Factory/BPSFactory.cs [hunk 3]
```diff
@@ -21,7 +18,7 @@ public override string GetFileName()
 
     public override string GetContent()
     {
-        StringBuilder sb = new StringBuilder();
+        StringBuilder sb = new();
```
Comment (johnwatson484) [inline FFCPayBatchGenerator/Factory/BPSFactory.cs:21]: Converted to newer convention for new

File: FFCPayBatchGenerator/Factory/FDMRFactory.cs [hunk 4]
```diff
@@ -54,7 +51,7 @@ public override string GetContent()
         return sb.ToString();
     }
 
-    private int GetSchemeCode(int schemeYear)
+    private static int GetSchemeCode(int schemeYear)
```
Comment (johnwatson484) [inline FFCPayBatchGenerator/Factory/FDMRFactory.cs:54]: Doesn't use member declarations so can be static.

File: FFCPayBatchGenerator/Factory/BaseBatchFactory.cs [hunk 5]
```diff
@@ -1,10 +1,4 @@
-﻿using System;
```
Comment (johnwatson484) [inline FFCPayBatchGenerator/Factory/BaseBatchFactory.cs:1]: There were several unused using declarations.  I've removed them all.

**Review by johnwatson484 — COMMENTED** [review]
Lump sums file spec is identical to BPS file spec.  Just with `SITI` references replaced with `LSES` plus different scheme and fund codes.

**Review by AbidemiAdio — APPROVED** [review]

**Pull request merged**